core/src/main/kotlin/pages/contentNodeProperties.kt

```diff
@@ -8,9 +8,16 @@ class SimpleAttr(val extraKey: String, val extraValue: String) : ExtraProperty<C
 
     companion object {
         fun header(value: String) = SimpleAttr("data-togglable", value)
+        fun togglableTarget(value: String) = SimpleAttr("data-togglable", value)
```

Comment:
If I understand correctly, these two functions do the same thing - what's the difference? When do I use the first one and when do I use the second?